pants

```diff
@@ -116,8 +121,8 @@ function determine_pants_version {
   fi
   pants_major_version="$(echo "${pants_version}" | cut -d '.' -f1)"
   pants_minor_version="$(echo "${pants_version}" | cut -d '.' -f2)"
-  if [[ "${pants_major_version}" -eq 1 && "${pants_minor_version}" -le 16 ]]; then
-    die "This version of the \`./pants\` script does not work with Pants <= 1.16.0. Instead,
+  if [[ "${pants_major_version}" -eq 1 && "${pants_minor_version}" -lt 23 ]]; then
+    die "This version of the \`./pants\` script does not work with Pants < 1.23.0. Instead,
 either upgrade your \`pants_version\` or use the version of the \`./pants\` script at
 https://raw.githubusercontent.com/pantsbuild/setup/3bb006e4a792ae83d7059ea76c0372ece7c1069d/pants."
```

Comment:
Should this now recommend a "more recent old version"?

Reply:
I'm not sure how to handle it. The most robust approach would be to have two checks. First check if < 16, and stick with this message. Then, check if >=16 and < 23, then do a new message.

Thoughts?

Reply:
It would probably be ok to walk a linked list of "try this script, it will recommend an older one, rinse repeat".